extensions/amp-story-interactive/0.1/test/test-amp-story-interactive-slider.js

```diff
@@ -78,5 +78,76 @@ describes.realWin(
         ampStorySlider.getRootElement().querySelector('input[type="range"]')
       ).to.not.be.null;
     });
+
+    it('should be disabled after the input event', async () => {
+      await ampStorySlider.buildCallback();
+      await ampStorySlider.layoutCallback();
+      const slider = ampStorySlider
+        .getRootElement()
+        .querySelector('input[type="range"]');
+      // simulates a change event, which is when the user releases the slider
+      slider.dispatchEvent(new CustomEvent('change'));
+      expect(slider.hasAttribute('disabled')).to.be.true;
+    });
+
+    it('should set the number displayed in the bubble to the same as the input', async () => {
+      await ampStorySlider.buildCallback();
+      await ampStorySlider.layoutCallback();
+      const slider = ampStorySlider
+        .getRootElement()
+        .querySelector('input[type="range"]');
+      const sliderBubble = ampStorySlider
+        .getRootElement()
+        .querySelector('.i-amphtml-story-interactive-slider-bubble');
+      slider.value = 30;
+      // simulates an input event, which is when the user drags the slider
+      slider.dispatchEvent(new CustomEvent('input'));
+      expect(sliderBubble.textContent).to.be.equal('30');
+    });
+
+    it('should display the percentage in the bubble when the user drags the slider', async () => {
+      await ampStorySlider.buildCallback();
+      await ampStorySlider.layoutCallback();
+      const slider = ampStorySlider
+        .getRootElement()
+        .querySelector('input[type="range"]');
+      slider.value = 30;
+      // simulates an input event, which is when the user drags the slider
+      slider.dispatchEvent(new CustomEvent('input'));
+      expect(
+        win
+          .getComputedStyle(ampStorySlider.getRootElement())
+          .getPropertyValue('--percentage')
+      ).to.be.equal('30%');
+    });
+
+    it('should show the bubble when the user drags the slider', async () => {
+      await ampStorySlider.buildCallback();
+      await ampStorySlider.layoutCallback();
+      const slider = ampStorySlider
+        .getRootElement()
+        .querySelector('input[type="range"]');
+      const sliderBubble = ampStorySlider
+        .getRootElement()
+        .querySelector('.i-amphtml-story-interactive-slider-bubble');
+      // simulates an input event, which is when the user drags the slider
+      slider.dispatchEvent(new CustomEvent('input'));
+      expect(sliderBubble.classList.contains('show')).to.be.true;
```

Comment:
Nit: Seems like this exists, lets use it: `expect(sliderBubble).to.have.class('show')`